marshmallow_recipe/metadata.py

```diff
@@ -27,6 +28,19 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return str(self.__values)
 
+    def __hash__(self) -> int:
+        if self.__hash is None:
+            h = 0
+            for key in sorted(self.__values):
```

Comment:
what is the reason to use `sorted`? result of `h ^= hash(key)` should not depend on the order of keys.

`a^b == b^a`

Reply:
Nice one. Thanks.